processor/src/stack/tests.rs

```diff
@@ -37,41 +35,6 @@ fn initialize() {
     assert_eq!(stack.helpers_state(), expected_helpers);
 }
 
-#[test]
-fn initialize_overflow() {
```

Comment:
Instead of removing, should we adapt the test to instead push the values 17, 18, and 19 after the call to `Stack::new()`?

Reply:
Ah, indeed. First I thought that this test only needed to test overflow table initialization, but now I see that we can also check its state after some manipulations. I'll rework it as you said.